src/idl_parser.cpp

```diff
@@ -918,10 +918,13 @@ CheckedError Parser::ParseField(StructDef &struct_def) {
   ECHECK(ParseType(type));
 
   if (struct_def.fixed) {
-    if (IsIncompleteStruct(type) || 
+    if (IsIncompleteStruct(type) ||
```

Comment:
this looks like formatting changes. I would run clang format on the whole repo and fix those in a separate PR, so that this PR can just focus on the logic since it is already complicated enough.